packages/office-ui-fabric-react/src/components/Autofill/Autofill.test.tsx

```diff
@@ -85,4 +85,18 @@ describe('Autofill', () => {
     expect(autofill.value).toBe('hel');
     expect(autofill.inputElement!.value).toBe('hello');
   });
+
+  it('value changes when updateValueInWillReceiveProps is passed in', () => {
+    const propsString = 'Updated';
+    const receivePropsUpdater = () => {
+      return propsString;
+    };
+    component = mount(<Autofill componentRef={autofillRef} suggestedDisplayValue="" updateValueInWillReceiveProps={receivePropsUpdater} />);
+
+    ReactTestUtils.Simulate.input(autofill.inputElement!, mockEvent('hel'));
+    component.setProps({ suggestedDisplayValue: 'hello' });
+
+    expect(autofill.value).toBe('Updated');
+    expect(autofill.inputElement!.value).toBe('Updated');
+  });
```

Comment:
Did this test fail before the change? It looks like it should have worked...

Reply:
Yep, just double checked. Line 99 will fail because it's making sure that the current value of the picker is matches what's returned by updateValueInWillReceiveProps, since before the actual value of the picker was never changed.

![image](https://user-images.githubusercontent.com/20801176/63115737-b1a04080-bf4c-11e9-809d-70ec58c1f8af.png)